File: elementary/three-words.py
#!/usr/bin/env python3

import re

def three_words(words):
    return re.search("[a-zA-Z]+ [a-zA-Z]+ [a-zA-Z]+", words) != None

if __name__ == '__main__':
    assert three_words("Mokus felmaszott fara") == True
    assert three_words("This squirrel 999 apple") == False
    assert three_words("9 8 7 6") == False
    assert three_words("little little little") == True
    assert three_words("Wow") == False

File: elementary/three-words.rb
#!/usr/bin/env ruby

def three_words(words)
    return (/[a-zA-Z]+ [a-zA-Z]+ [a-zA-Z]+/ =~ words) != nil
end

if __FILE__ == $0
    raise unless three_words("Mokus felmaszott fara") == true
    raise unless three_words("This squirrel 999 apple") == false
    raise unless three_words("9 8 7 6") == false
    raise unless three_words("little little little") == true
    raise unless three_words("Wow") == false
end
